java/CMakeLists.txt

```diff
@@ -103,23 +108,26 @@ if(NOT WIN32)
   #   OUTPUT_NAME ${CMAKE_CURRENT_SOURCE_DIR}/src/main/c++/jni_spark_vw_generated.h)
   # add_dependencies(javaheaders javacompile)
 
+  SET(mvn_args -B verify)
+  if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+      set(JAVA_INSTALL_PATH /usr/lib)
+      # the integration tests don't currently pass on mac
+      LIST(APPEND mvn_args "-DskipTests")
+  else()
+      set(JAVA_INSTALL_PATH /Library/Java/Extensions)
```

Comment:
This actually appears to be the opposite logic as what it should be - if we're on Darwin the path should be `/Library/Java/Extensions` IIUC.  I didn't want to make an irrelevant change here but I'd be glad to tack it on.

Reply:
Yeah it does seem backwards, let's do it in a separate PR though